streams/shared/src/main/scala/zio/stream/ZSink.scala

```diff
@@ -921,19 +922,15 @@ object ZSink extends ZSinkPlatformSpecificConstructors {
     z: => S
   )(contFn: S => Boolean)(f: (S, In) => S)(implicit trace: Trace): ZSink[Any, Nothing, In, In, S] =
     ZSink.suspend {
-      def foldChunkSplit(z: S, chunk: Chunk[In])(
-        contFn: S => Boolean
-      )(f: (S, In) => S): (S, Chunk[In]) = {
```

Comment:
No need to pass `contFn` and `f` here are they are the same than the ones already present in the wrapping context.
Not having to pass them here makes it more explicit that you're actually using the "global" ones